source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDateTextField.mm

```diff
@@ -41,7 +41,15 @@ - (instancetype)initWithTimeDateInput:(std::shared_ptr<BaseInputElement> const &
         _encodeFormatter = [[NSDateFormatter alloc] init];
         [self configDateFormatter:_encodeFormatter formatterStyle:dateStyle];
 
-        UIDatePicker *picker = [bundle loadNibNamed:@"ACRDatePicker" owner:self options:nil][0];
+        UIDatePicker *picker = nil;
+        if (@available(iOS 14.0, *)) {
```

Comment:
Looks like this is available since iOS 13.4, should we enable it back to that?

Reply:
PickerStyle property is added in 13.4. The PickerStyleInline is added in 14.0